routes/api.php

```diff
@@ -55,6 +56,7 @@
         Route::delete('statuses/{id}', [StatusController::class, 'destroy']);
         Route::put('statuses/{id}', [StatusController::class, 'update']);
         Route::post('support/ticket', [SupportController::class, 'createTicket']);
+        Route::apiResource('statuses/{statusId}/tags', StatusTagController::class);
```

Comment:
Bad practice in this application. Status tags endpoints are of middlewaretype authenticated AND semiguest (for my understanding, index & show shall be available for guests as well as authenticated users. ➜ semiguest)